packages/preview/universal-cau-thesis/0.1.0/ref/template.typ

```diff
@@ -0,0 +1,361 @@
+#import "./booktab.typ": *
+#import "@preview/codly:0.2.0": *
+#import "@preview/codelst:2.0.1": sourcecode
+#import "./acronyms.typ": acro, usedAcronyms, acronyms
+
+#let project(
+  kind: "硕士",
+  title: "中国农业大学论文模板",
+  abstract_en: [],
+  abstract_zh: [],
+  title_en:[],
+  title_zh:[],
```

Comment:
The convention in Typst packages is to use kebab-case names (it is only a convention, if you prefer to use snake_case, just tell me).

Reply:
I have already format the code into kebab-case